### PR TITLE
publish tags

### PR DESCRIPTION
**Pull request by antony**
WIP for now, clean up our debugging, and prepare to set version in the commit message for the template repository.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [ ] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

**Comment by antony** [comment]
Before we try to get the `npm_version_number` into this PR, we need to consider if this is the correct version number to include. Updating kit components without updating the cli obviously would not update this release number, and all of the kit component versions would be listed in the `package.json` anyway. So I wonder if this becomes more of a date, or something, or a reference to a build workflow run, instead.

File: .github/workflows/update_template.yml [hunk 1]
```diff
@@ -51,9 +49,8 @@ jobs:
       - name: Publish
         working-directory: ${{ github.workspace }}/template
         run: |
-          touch "generated-$(date +%s)"
           git config user.email "noreply@svelte.dev"
           git config user.name "[bot]"
           git add -A
-          git commit -m "version $npm_package_version"
+          git commit -m "version ${{ github.ref_name }}"
```
Comment (Rich-Harris) [inline .github/workflows/update_template.yml:55]: maybe this?

```suggestion
          git commit -m "create-svelte version ${{ github.ref_name }}"
```

File: .github/workflows/update_template.yml [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
 
 on:
-  workflow_run:
-    workflows: [Release]
-    types:
-      - completed
-  workflow_dispatch:
+  push:
+    tags:
+      - create-svelte@*
```
Comment (pngwn) [inline .github/workflows/update_template.yml:6]: Are we actually tagging? I thought we weren't? This will never run if we aren't.

**Review by pngwn — COMMENTED** [review]
Small comment, looks fine otherwise.

File: .github/workflows/update_template.yml (hunk 2 above)
Comment (antony) [reply .github/workflows/update_template.yml:6]: That's the other part of this PR. According to changesets docs, they always tag on publish, but unless you push those tags, they get discarded.

Comment (pngwn) [reply .github/workflows/update_template.yml:6]: Are we going to start publishing with changesets then instead of doing it manually?

Comment (Rich-Harris) [reply .github/workflows/update_template.yml:6]: we already do publish with changesets, no?

File: package.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
 		"check": "pnpm -r check",
 		"lint": "pnpm -r lint",
 		"format": "pnpm -r format",
-		"release": "pnpm publish --tag=next --filter=\"@sveltejs/*\" --filter=\"create-svelte\"",
+		"release": "pnpm publish --tag=next --filter=\"@sveltejs/*\" --filter=\"create-svelte\" && git push --follow-tags",
```
Comment (Rich-Harris) [inline package.json:12]: is this the right change @antony / @pngwn?

```suggestion
		"release": "pnpm build -r && changesets publish && git push --follow-tags",
```

Comment (antony) [reply package.json:12]: I think you call `changeset` via pnpm i.e. `pnpm changeset version` rather than directly.

I'll update the PR.

Comment (antony) [reply package.json:12]: Ok done, one question, why the `pnpm build -r`. We have already built the monorepo at this point via `pnpm build` which is the previous step in `release.yaml`.

I think we'd be safe to remove it from here.

Comment (Rich-Harris) [reply package.json]: Pretty sure the publish step was building, so I was just copying what was already happening. If that means we've been building twice and now we only need to build once, then great

**Review by Rich-Harris — APPROVED** [review]

**Pull request merged**